src/App.tsx

```diff
@@ -6,15 +6,16 @@ import './App.less';
 import Home from './containers/home/Home';
 import Signup from './containers/signup/Signup';
 import Login from './containers/login/Login';
+import ForgotPassword from './containers/forgot-password/ForgotPassword';
```

Comment:
```suggestion
import ForgotPassword from './containers/forgotPassword';
```
If you rename the directory and change the file name to `index.tsx` you can import with this shortened file path.